das-dashboard/backend/controllers/query_controllers.py

```
@@ -0,0 +1,123 @@
+        )
+    except json.JSONDecodeError:
+        await websocket.send_json(
+            {
+                "execution_id": execution_id,
+                "status": "error",
+                "message": "Invalid JSON received from the command router stream.",
+            }
+        )
+
+
+def _proxy_json_content(response : Response):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_proxy_json_content`

(ANN202)